app/policies/follow_policy.rb

```diff
@@ -13,6 +13,7 @@ def visible_attributes(all)
   # Override to user follower instead of user
   def is_owner? # rubocop:disable Style/PredicateName
     return false unless user && record.follower_id == user.id
+    return false if user.has_role?(:banned)
```

Comment:
The user still owns their own follows even if they're banned.

This should probably be up in `def create?` as `is_owner? && !user.has_role?(:banned)`